ConwaysGameOfLife/Generator.cs

```diff
@@ -23,6 +25,23 @@ rules checked
             output
             universe
             rules
+            inputparser
+            outputformatter
         */
+        Universe _universe;
+
+        
+
+        public Generator(Universe universe)
+        {
+            _universe = universe;
+            
+        }
+
+        public void DisplayUniverse(IOutput output, OutputFormatter formatter)

```

Comment:
FYI C# has abstractions of Console.In and Console.Out called [Textwriter ](https://docs.microsoft.com/en-us/dotnet/api/system.io.textwriter?view=net-5.0)and [Textreader](https://docs.microsoft.com/en-us/dotnet/api/system.io.textreader?view=net-5.0) which saves you from having to create your own interface for them